### PR TITLE
Create apple-rachanee-saengkrajai.md

### PR DESCRIPTION
**Pull request by RachaneeSaeng**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=1058) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=1058) before we can accept your contribution.<br/><br/>:x: RachaneeSaeng [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=1058)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=1058) it.</sub>

**Review by Arhell — APPROVED** [review]

File: input/community/speakers/apple-rachanee-saengkrajai.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+Title: Apple Rachanee Saengkrajai
+Pronouns: She
```
Comment (rprouse) [inline input/community/speakers/apple-rachanee-saengkrajai.md:3]: I think this needs to be She/Her, but the build will tell us.

**Review by rprouse — APPROVED** [review]

File: input/community/speakers/apple-rachanee-saengkrajai.md (hunk 1 above)
Comment (rprouse) [reply input/community/speakers/apple-rachanee-saengkrajai.md:3]: Yes, build failed because of this.

Comment (rprouse) [reply input/community/speakers/apple-rachanee-saengkrajai.md]: I made the change, waiting for the build.

**Pull request merged**